packages/dashboard/src/components/Wizard/Wizard.styles.ts

```diff
@@ -18,14 +84,64 @@ export const getWizardStyles = (props: IWizardStyleProps): IWizardStyles => {
       borderRight: `1px solid ${props.theme.semanticColors.bodyDivider}`,
       overflowY: 'auto'
     },
+    contentAnimSection: {

```

Comment:
Since this isn't animating let's rename this. Maybe call it contentSectionContainer or something.